verticapy/performance/vertica/__init__.py

```diff
@@ -15,3 +15,6 @@
 permissions and limitations under the License.
 """
 from verticapy.performance.vertica.qprof import QueryProfiler
+from verticapy.performance.vertica.collection_metadata import CollectionMetadata
+from verticapy.performance.vertica.step00_vertica_version import Step00VerticaVersion
```

Comment:
I really don't like the name, will we create a file for each step? I don't think we will even implement all of them.